code/game/machinery/telecomms/machines/server.dm

```diff
@@ -72,7 +72,8 @@
 				log.parameters["name"] = signal.data["name"]
 				log.parameters["realname"] = signal.data["realname"]
 
-				log.parameters["uspeech"] = signal.data["languages"] & HUMAN //good enough
+				//log.parameters["uspeech"] = signal.data["languages"] & HUMAN //good enough
```

Comment:
universal speech.
it's an old hack, this says that if any one of their languages is HUMAN, everyone can understand them over the radio (this is dumb)